File: LeetCode/1704. Determine if String Halves Are Alike/1704. Determine if String Halves Are Alike.py
class Solution:
    def halvesAreAlike(self, s: str) -> bool:
        s = s.lower()
        l_str = int(len(s)/2)
        vow = ['a','e','i','o','u']
        cnt1 = 0
        cnt2 = 0
        for i in range(l_str):
            if(s[i] in vow):
                cnt1 += 1
            if(s[i+l_str] in vow):
                cnt2 += 1
                
        if(cnt1 == cnt2):
            return True
        else:
            return False

File: LeetCode/1704. Determine if String Halves Are Alike/1704. Determine if String Halves Are Alike.java
class Solution {
    public boolean halvesAreAlike(String s) {
        s = s.toLowerCase();
        String a = s.substring(0,s.length()/2);
        String b = s.substring(s.length()/2,s.length());
        int cnt1=0;
        int cnt2=0;
        for(int i=0;i<a.length();i++){
            if(a.charAt(i)=='a'||a.charAt(i)=='e'||a.charAt(i)=='i'||a.charAt(i)=='o'||a.charAt(i)=='u'){
             cnt1++;
            }
            if(b.charAt(i)=='a'||b.charAt(i)=='e'||b.charAt(i)=='i'||b.charAt(i)=='o'||b.charAt(i)=='u'){
             cnt2++;
            }
        }
        if(cnt1==cnt2)
            return true;
        else
            return false;

    }
}